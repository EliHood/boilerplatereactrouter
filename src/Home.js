import React from 'react';

const Home = () => (
    <div className="main">
        <h2>Home</h2>
    </div>
)

export default Home;