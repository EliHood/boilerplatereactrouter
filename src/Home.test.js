import React from 'react';
import ReactDOM from 'react-dom';
import { shallow, mount} from 'enzyme';
import Home from './Home';
import toJson from 'enzyme-to-json';
describe('Should render <Home/> Component', () => {
    it('should render <Home/> Component', () => {
        const wrapper = shallow(<Home/>)
        expect(wrapper.find('div')).toHaveLength(1);
    })

    it('should match <Home/> snapshot', () => {
        const wrapper = shallow(<Home/>)
        expect(toJson(wrapper)).toMatchSnapshot();
    })


    it('should find h1', () => {
        const wrapper = shallow(<Home/>);
        expect(wrapper.find('h2').text()).toBe('Home')
    })

})