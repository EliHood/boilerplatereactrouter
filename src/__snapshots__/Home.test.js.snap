// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render <Home/> Component should match <Home/> snapshot 1`] = `
<div
  className="main"
>
  <h2>
    Home
  </h2>
</div>
`;

exports[`Should render <Home/> Component should match snapShot 1`] = `
<div
  className="main"
>
  <h2>
    Home
  </h2>
</div>
`;
